### ISSUE TITLE
Resend notifications after a period of time

### ISSUE THREAD

**Issue by mthjones** [mthjones/pierre#2]

**Comment by ekelundh** [comment]
I'll work on this for now, seems easy enough to implement. Add a row in the DB for 'processed time' and on each 15 minute checkup check the processed_prs table? Not sure.

**Comment by ekelundh** [comment]
I thought the close and comment meant close the comment tab :(

**Comment by mthjones** [comment]
Yeah, that sounds good. Check out the chrono library to handle dates and
times.

Maybe re-notify at like...10am every day after the day it was created?

**Comment by ekelundh** [comment]
Sounds good. Would be configurable in later iterations if we want as well.
